scripts/build_food_db.py

```
@@ -63,35 +65,37 @@ def load_source_data(self) -> tuple[List[Dict], List[Dict]]:
         else:
             print("  📊 Loading OFF from single file")
-            adapter = OFFAdapter()
+            off_adapter = OFFAdapter()
 
         try:
-            off_data = list(adapter.normalize())
+            off_data = list(off_adapter.normalize())
             print(f"  ✅ OFF: {len(off_data)} records")
-        except Exception as e:
-            print(f"  ❌ OFF error: {e}")
+        except (OSError, ValueError, UnicodeDecodeError) as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)